debugging/usb.md

```diff
@@ -0,0 +1,27 @@
+# Debugging USB Communications
+
+### Introduction to USB
+
+For an in-depth description of how USB works, read [about a code deploy example](https://github.com/tessel/onboarding/blob/master/T2-TECHNICAL-OVERVIEW.md#a-code-deploy-example). Otherwise, the TL;DR is that USB communication from the CLI gets sent to the atmel samd21 coprocessor which then routes the packets over SPI to a daemon running on OpenWRT which then passes it on to another daemon whose only job is to create processes requested over USB and pipe the standard streams back to it. If that description caused your eyes to glaze over, that's okay because the only thing you need to take out of that is that for USB comms to be working, you need:
```

Comment:
hmm if you know it will cause eyes to glaze over (mine did by the second sentence) think about rewriting it/breaking it up.